### PR TITLE
Added more conversion types

### PR DESCRIPTION
**Pull request by namalu**
Added conversion types and tests for:
* Body mass (weight)
* Temperature
* Blood oxygen saturation
* Respiratory rate

### CODE REVIEW

File: Sources/Configuration/DefaultObservationFactoryConfig.json [hunk 1]
```diff
@@ -94,5 +94,84 @@
...
+                }
+            ]
+        },
+        "valueQuantity": {
+            "unit" : "%",
+            "system" : "http://unitsofmeasure.org",
+            "code" : "%"
+        }
+    },
+    "HKQuantityTypeIdentifierBodyMass": {
```
Comment (LiquidPT) [inline Sources/Configuration/DefaultObservationFactoryConfig.json:119]: Is this what it's called in HK? Seems weird to be mixing mass and weight terms

File: Sources/Configuration/DefaultObservationFactoryConfig.json [hunk 2]
```diff
@@ -94,5 +94,84 @@
...
+            "coding": [
+                {
+                    "system": "http://loinc.org",
+                    "code": "8310-5",
+                    "display": "Body temperature"
+                }
+            ]
+        },
+        "valueQuantity": {
+            "unit" : "degC",
```
Comment (LiquidPT) [inline Sources/Configuration/DefaultObservationFactoryConfig.json:156]: Should we standardize on metric or SAE? I see a mix in this file. Or is this a measurement by measurement decision?

File: Sources/Configuration/DefaultObservationFactoryConfig.json (hunk 1 above)
Comment (namalu) [reply Sources/Configuration/DefaultObservationFactoryConfig.json:119]: HKQuantityTypeIdentifierBodyMass is the actual HealthKit type for body weight. [Here](https://developer.apple.com/documentation/healthkit/hkquantitytypeidentifierbodymass?language=objc) is the documentation.

File: Sources/Configuration/DefaultObservationFactoryConfig.json (hunk 2 above)
Comment (namalu) [reply Sources/Configuration/DefaultObservationFactoryConfig.json:156]: I updated everything to normalize around metric units.

**Pull request merged**